crates/typst/src/layout/page.rs

```diff
@@ -41,9 +41,11 @@ use crate::visualize::{Color, Paint};
 #[elem(Construct)]
 pub struct PageElem {
     /// A standard paper size to set width and height.
+    ///
+    /// This is just a shorthand for setting `width` and `height` and, as such,
+    /// cannot be retrieved in a context expression.
     #[external]
     #[default(Paper::A4)]
-    #[ghost]
     pub paper: Paper,
```

Comment:
Does removing `#[ghost]` have the effect of prohibiting access from context? (Just curious.)

Reply:
It's not in context anyway. Since it's external and just a shorthand. The ghost did nothing.